src/js/game/SoloGame.js

```diff
@@ -10,6 +12,21 @@ class SoloGame extends ClientGame {
     super(emitter, actions)
   }
 
+  start() {
+    // Duplicated from Game.js @TODO set this up properly
```

Comment:
Oh this is something that I had a question about, it's pulling that code from Game (Game > ClientGame > SoloGame), I don't know if there's a clean way to do that (you'd use super.start() for the parent, though I don't believe super.super.start() works - not for the lack of trying, mind you)

Reply:
This file actually has access to `start()` via `this.start()` because of the inheritance chain you described. `super` is only useful when you have an inherited method that you want customized. You can call `super.methodName()` and then write more code to happen after that.

Reply:
Added method startSolo() to Game to work around inheritance issue. Added/customized reset method of SoloGame to reset pathHelper. Selecting new game after the end of a prior game properly resets the board for a SoloGame instance.